### ISSUE TITLE
Title bar height option

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#5468]
After the latest update, title bar height increased and 
![Screen Shot 2022-08-05 at 10 57 34 AM](https://user-images.githubusercontent.com/60490540/183032326-cb2bb417-6387-4c02-b918-86740c78d3e4.png)
no option to control its height.

I always, use my vscode `window.zoomLevel: 2` and **title bar** wasn't changing according to **zoomLevel**.

If, this is a new accessibility feature then, we should have an option to control **title bar** height.
